File: Python/1411.py
N = int(input())
arr = []
for i in range(N-1) : arr.append(int(input()))

for j in range(1, N+1):
    cnt = 0
    for i in range(N-1):
        cnt += 1
        if j==arr[i]:
            cnt = 0
            break
    if cnt==N-1:
        print(j)
        break

File: C++/1411.cpp
#include <iostream>

using namespace std;

int main(){
	int N, cnt;
	cin >> N;
	int arr[N-1];
	for(int i = 0; i < N-1; i++){
		cin >> arr[i];
	}
	
	for(int j = 1; j <= N; j++){
		cnt = 0;
		for(int i = 0; i < N-1; i++){
			cnt++;
			if(j==arr[i]){
				cnt = 0;
				break;
			}
		}
		if(cnt==N-1){
			cout << j;
			break;
		}
	}
}